nautobot/core/tests/test_releases.py

```diff
@@ -99,6 +99,7 @@ def test_pre_releases(self, mock_cache_get: Mock, mock_cache_set: Mock, mock_req
             "https://localhost/unittest/releases",
             headers={"Accept": "application/vnd.github.v3+json"},
             proxies=settings.HTTP_PROXIES,
+            timeout=15,
```

Comment:
Just for my own curiosity, why did we add the time out here?

Reply:
Complaint by pylint that `requests` can potentially hang forever if there's no timeout provided. We already did the same change in `next` in #3027.